snapcraft.yaml

```diff
@@ -751,7 +751,6 @@ parts:
       - spice-protocol
       - spice-server
     source: https://github.com/qemu/qemu
-    source-commit: 6bbce8b464206e6622216b62841cb3e953d56eb8 # v8.0.5
```

Comment:
Could we keep that but add `source-submodules: []` to avoid pulling submodules automatically?

Reply:
Yeah worth a try indeee but ill merge this now to unblock us